src/Console/Commands/DbCreateCommand.php

```diff
@@ -15,7 +15,8 @@ class DbCreateCommand extends Command
      * @var string
      */
     protected $signature = 'db:create
-                                {database? : Database to create. If omitted, name from .env will be used.}';
+                                {database? : Database to create. If omitted, name from .env will be used.}
+                                {--allow-existing : Exit successfully even if the database already exists.}';
```

Comment:
I'm not at all in love with this option name, so feel free to suggest an alternative.